clusters/app.ci/pr-reminder/pr-reminder-config.yaml

```diff
@@ -134,3 +134,16 @@ data:
         repos:
         - openshift/windows-machine-config-operator
         - openshift/windows-machine-config-bootstrapper
+      - teamMembers:
+        - petr-muller
+        - wking
+        - LalatenduMohanty
+        - PratikMahajan
+        - jottofar
```

Comment:
@wking @LalatenduMohanty @PratikMahajan @jottofar let me know if you want me to drop you from here